src/scripts/components/BillableIcon.tsx

```diff
@@ -4,26 +4,11 @@ import styled from '@emotion/styled'
 
 import * as color from '../@toggl/style/lib/color';
 
-interface Theme {
```

Comment:
Was this the start of a react theme function? I don't think it was being used so removed it :eyes:

Reply:
I don't recall why this was added

Reply:
I think it's copypaste from the web codebase

Reply:
Ok, sounds like it's safe to remove :hugs: